appengine/go11x/pubsub/authenicated_push/main.go

```diff
@@ -112,6 +112,12 @@ func (a *app) receiveMessagesHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	// IMPORTANT: you should validate claim details not covered by signature
+	// and audience verification above, including:
+	//   - Ensure that `payload.Claims["email"]` is equal to the expected service
+	//     account set up in the push subscription settings.
+	//   - Ensure that `payload.Claims["email_verified"]` is set to true.
```

Comment:
Should we check this directly in the code, rather than only in the comment?

Reply:
I don't believe we have the service account email here since it's set up by the user to run these tests.

Reply:
I meant the `email_verified` check, not the `email` itself. Unless I'm missing something?

Reply:
We could also use a placeholder email, like we do above:

```
	// Please change http://example.com to match with the value you are
	// providing while creating the subscription.
```

Reply:
Updated, PTAL.